studio/components/interfaces/SignIn/SignInWithGitHub.tsx

```diff
@@ -24,7 +24,7 @@ const SignInWithGitHub = () => {
             process.env.NEXT_PUBLIC_VERCEL_ENV === 'preview'
               ? location.origin
               : process.env.NEXT_PUBLIC_SITE_URL
-          }${BASE_PATH}${getReturnToPath()}`,
+          }${BASE_PATH}${getReturnToPath('/')}`,
```

Comment:
This is to force the user to be redirected to the sign in page, where he would be checked if he has MFA.